gallery/rediscache-insights/Azure Monitor.json

```diff
@@ -3,11 +3,11 @@
 	"version": "TemplateGallery/1.0",
 	"categories": [
 		{
-			"id": "Insights",
-			"name": "Insights",
+			"id": "RedisCache",
+			"name": "Azure Cache for Redis Insights",
 			"templates": [
 				{
-					"id": "Workbooks/Insights/RedisCache",
+					"id": "Workbooks/RedisCache/AtScale",
```

Comment:
what does 'AtScale' mean? These 3 files seems to be in 3 different locations.  Whats the significance of ID being different in these two files vs the one inside microsoft-cache-redis.json?

Reply:
The "AtScale" naming refers to workbooks that appear in the Azure Monitor menu and Insights Hub explorer in Azure Portal. Those workbooks give an overview of all resources for a service:

![image](https://user-images.githubusercontent.com/4934473/115616158-d91ce400-a2bd-11eb-83c3-0e349d9f66b2.png)

Clicking on an individual resource name in that workbook jumps to the "AtResource" workbook that drills down into details for the particular resource:

![image](https://user-images.githubusercontent.com/4934473/115616283-08335580-a2be-11eb-933b-b1b000cdb2ab.png)

The motivation to change the path to "AtScale" is to align with other similar workbooks:

- _Workbooks\ADXCluster\\**AtScale**\Insights.workbook_
- _Workbooks\Activity Log\\**AtScale**\Insights.workbook_
- _Workbooks\EventGrid\\**AtScale**\Domains\Insights.workbook_
- _Workbooks\EventGrid\\**AtScale**\Topics\Insights.workbook_
- _Workbooks\EventHub\\**AtScale**\Insights.workbook_
- _Workbooks\LogAnalytics Workspace\\**AtScale**\Workspace Insights At Scale.workbook_
- _Workbooks\RedisCache\\**AtScale**\Insights.workbook_
- _Workbooks\ServiceBus\\**AtScale**\Insights.workbook_